L1Trigger/L1TTrackMatch/src/METTrackTransform.cc

```diff
@@ -0,0 +1,123 @@
+#include <cmath>
+#include "L1Trigger/L1TTrackMatch/interface/METTrackTransform.h"
+
+void METTrackTransform::GenerateLUTs(){
+    phi_quadrants = generate_phi_slice_LUTs (L1TkEtMissEmuAlgo::N_quadrants_);
+    phi_shift     = generate_phi_slice_LUTs (L1TkEtMissEmuAlgo::N_sectors_);
+}
+
+InternalEtWord METTrackTransform::TransformTrack(TTTrack< Ref_Phase2TrackerDigi_ >& track_ref, float PV){
```

Comment:
Same comment about possibly being redundant.

Reply:
Fixed in latest commit